### PR TITLE
C wrapper

### PR DESCRIPTION
**Pull request by huskar-t**
taos_reset_current_db
taos_validate_sql
taos_is_update_query

@sangshuduo @zitsen

### CODE REVIEW

File: wrapper/taosc_test.go [hunk 1]
```diff
@@ -141,3 +143,121 @@ func TestTaosQueryA(t *testing.T) {
...
+			},
+			want: 0,
+		},
+		{
+			name: "TSC_SQL_SYNTAX_ERROR",
+			args: args{
+				taosConnect: conn,
+				sql:         "slect 1",
+			},
+			want: 0x0216,
```
Comment (sangshuduo) [inline wrapper/taosc_test.go:222]: we should avoid using the magic number

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by zitsen — APPROVED** [review]
LGTM

File: wrapper/taosc.go [hunk 2]
```diff
@@ -155,3 +155,20 @@ func TaosQueryA(taosConnect unsafe.Pointer, sql string, caller cgo.Handle) {
...
+// TaosResetCurrentDB void taos_reset_current_db(TAOS *taos);
+func TaosResetCurrentDB(taosConnect unsafe.Pointer) {
+	C.taos_reset_current_db(taosConnect)
+}
+
+// TaosValidateSql int taos_validate_sql(TAOS *taos, const char *sql);
+func TaosValidateSql(taosConnect unsafe.Pointer, sql string) int {
+	cSql := C.CString(sql)
+	defer C.free(unsafe.Pointer(cSql))
+	return int(C.taos_validate_sql(taosConnect, cSql))
```
[the remark this answers is not in the document]
Comment (huskar-t) [reply wrapper/taosc.go:168]: I plan to return the wrapper function of this package to be consistent with C. Due to historical issues, the TaosConnect interface is an exception.

**Pull request merged**